### PR TITLE
Main component refactoring

### PR DESCRIPTION
**Pull request by damnko**
Hi, I partially refactored only the `thead` part of the table, could you confirm me that the approach is ok so I can continue on the other parts of the component?
Thanks

### CODE REVIEW

**Comment by nnixaa** [comment]
@LexZhukov could you take a look please?

File: src/ng2-smart-table/components/thead/cells/title/title.component.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Component, Input, Output, EventEmitter } from '@angular/core';
 
-import { DataSource } from '../../lib/data-source/data-source';
-import { Column } from '../../lib/data-set/column';
+import { DataSource } from '../../../../lib/data-source/data-source';
+import { Column } from '../../../../lib/data-set/column';
 
 @Component({
   selector: 'ng2-smart-table-title',
```
Comment (LexZhukov) [inline src/ng2-smart-table/components/thead/cells/title/title.component.ts]: - please, rename `title.scss` to `title.component.scss` 😃

**Comment by damnko** [comment]
Thanks for your feedback, I will fix everything when I will update the PR with the full component refactored. I was mainly looking for a feedback on the approach, but I will consider it ok so far ;)
Thanks

**Comment by LexZhukov** [comment]
@damnko, your approach looks good:)

**Pull request closed without merging**